src/components/landing/SliderForEvents/SliderForEvents.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import PropTypes from "prop-types";
+import Image from "../../../mockup-images/football-g10e9bda93_1920.jpg";
+import EventCard from "../EventCard/EventCard";
+import "./SliderForEvents.scss";
+
+const SliderForEvents = ({ events }) => {
+  const settings = {
```

Comment:
As this is a constant, which doesn't depend on any external variable, please extract it from component to not recreate it on each rerender

Reply:
Moved to settingsForSlider.js. 
DONE